docs/server/source/appendices/run-with-ansible.md

```diff
@@ -2,8 +2,8 @@
 
 **NOT for Production Use**
 
-You can use the following instructions to deploy a BigchainDB node for
-dev/test using Ansible. Ansible will setup a BigchainDB node along with
+You can use the following instructions to deploy a single or multi node
```

Comment:
please add minimum recommended resources for each use case

Reply:
I will look into it.